lib/Braintree/Version.php

```diff
@@ -12,7 +12,7 @@ class Version
      */
     const MAJOR = 3;
     const MINOR = 25;
-    const TINY = 0;
+    const TINY = 1;
```

Comment:
The release process takes care of upgrading the constant, so can this change be reverted?

Reply:
Agreed. This change should be removed.